changelog.d/20250411_105010_8730430+m1yag1.md

```diff
@@ -0,0 +1,5 @@
+### Enhancements
+
+* Added the `--run-managers` and `--run_monitors` options to the `globus flows create` command
+  and the `globus flows update` command.
+* Added "Run Managers" and "Run Monitors" to the output of the `globus flows show` command.
```

Comment:
```suggestion
* Added `--run-manager` and `--run-monitor` options to `globus flows create`.
* Added `--run-managers` and `--run-monitors` options to `globus flows update`.
* Added "Run Managers" and "Run Monitors" to the output of `globus flows show`.
```

I'm not a big fan of the `--run-manager`/`--run-managers` dichotomy in this PR -- I see this isn't the first case of this -- but if it's going to exist we need to be careful to document it accurately.